_posts/2022-04-11-1.15-announcement.md

```diff
@@ -0,0 +1,375 @@
+---
+layout: post
+title:  "Announcing the Release of Apache Flink 1.15"
+subtitle: ""
+date: 2022-04-11T08:00:00.000Z
+categories: news
+authors:
+- yungao:
+  name: "Yun Gao"
+  twitter: "YunGao16"
+- joemoe:
+  name: "Joe Moser"
+  twitter: "JoemoeAT"
+
+---
+
+Thanks to our well-organized, kind, and open community, Apache Flink continues 
+[to grow](https://www.apache.org/foundation/docs/FY2021AnnualReport.pdf) as a 
+technology. We are and remain one of the most active projects in
+the Apache community. With release 1.15, we are proud to announce a number of 
+exciting changes.
+
+One of the main concepts that makes Apache Flink stand out is the unification of 
+batch (aka bounded data) and streaming (aka unbounded data) processing. A lot of 
+effort went into this in the last releases but we are only getting started there. 
+Apache Flink is not only growing when it comes to contributions and users, it is 
+also growing out of the original use cases and personas. Like the whole industry, 
+it is moving more towards business/analytics use cases that are implemented as 
+low-/no-code. The feature that represents the most within the Flink space is 
+Flink SQL. That’s why its popularity continues to grow. 
+
+Apache Flink is considered an essential building block in data architectures.  It 
+is included with other technologies to drive all sorts of use cases. New ideas pop 
+up, existing technologies establish themselves as standards for solving some aspects 
+of a problem. In order to be successful, it is important that the experience of 
+integrating with Apache Flink is as seamless and easy as possible. 
+
+In the 1.15 release the Apache Flink community made significant progress across all 
+these areas. Still those are not the only things that made it into 1.15. The 
+contributors improved the experience of operating Apache Flink by making it much 
+easier and more transparent to handle checkpoints and savepoints and their ownership, 
+making auto scaling more seamless and complete, by removing side effects of use cases 
+in which different data sources produce varying amounts of data, and - finally - the 
+ability to upgrade SQL jobs without losing the state. By continuing on supporting 
+checkpoints after tasks finished and adding window table valued functions in batch 
+mode, the experience of unified stream and batch processing was once more improved 
+making hybrid use cases way easier. In the SQL space, not only the first step in 
+version upgrades have been added but also JSON functions to make it easier to import 
+and export structured data in SQL. Both will allow users to better rely on Flink SQL 
+for production use cases in the long term. To establish Apache Flink as part of the 
+data processing ecosystem we improved the cloud interoperability and added more sink 
+connectors and formats. And yes we enabled a Scala-free runtime 
+([the hype is real](https://flink.apache.org/2022/02/22/scala-free.html)).
```

Comment:
```suggestion
In the 1.15 release the Apache Flink community made significant progress across multiple areas,
which will be discussed in this post! 
```

Reply:
@infoverload why is it bad to have an overview on the very top?

* it quickly provides an overview on what has happened.
* it is not bad to mention things a couple of times.